utils/generate-module-api-doc.rb

```diff
@@ -160,7 +160,7 @@ def is_func_line(src, i)
 end
 
 # Populate the 'since' map (name => version) if we're in a git repo.
-require File.dirname(__FILE__) ++ '/module-api-since.rb'
+require "./" ++ File.dirname(__FILE__) ++ '/module-api-since.rb'
```

Comment:
Why was this changed?

Reply:
I don't know ruby that well, but with this change I can call the script regardless of my current directory.

Reply:
File.dirname(__FILE__) is apparently a _relative_ path.

Reply:
Ok